src/components/Toolbar/Toolbar.jsx

```diff
@@ -11,13 +15,16 @@ import homeLogo from '../../assets/homeLogo.png';
 const DEFAULT_CLASSES = ['main-nav', 'user-actions'];
 const MEDIUM_BREAKPOINT = 960;
 
+
 function Toolbar() {
+  
   /************************************
    * State
    ************************************/
 
   const [isOpen, setIsOpen] = useState(false);
   const [classes, setClasses] = useState(DEFAULT_CLASSES);
+  const isMobile = useMediaQuery({query: '(max-width: 960px)'})
```

Comment:
can you make that 960px a constant at the top? under that name MEDIUM_BR would help us keep track of it if we ever need to change it down the road.

Reply:
Am I keeping the useMediaQuery and moving it at the top ? Or are you saying make it const isMobile = 960px

Reply:
but maybe those ticks can't be used with useMediaQuery? If that's not possible then just add a semicolon to the end of the line

Reply:
> <img alt="Screen Shot 2020-04-21 at 8 44 12 PM" width="50" src="https://user-images.githubusercontent.com/30677661/79928688-9954bf80-8411-11ea-873e-7879595bed74.png">
> 
> Where's the logo coming from? I don't think the image is right if it's cut off like this. We might need to get you a better image

That image came from the figma. If you have a better one please provide it.

Reply:
👍